explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -146,6 +148,19 @@ object ObsTabTiles:
     acquisition: Option[NonEmptyList[Offset]]
   )
 
+  // TODO Move to core
+  extension (om: ObservingMode)
+    def centralWavelength: Option[Wavelength] =
```

Comment:
We have a `CentralWavelength` type alias, do we want to return that instead?

Reply:
Yeah, I think that would be better, I'll change it